Dockerfile

```diff
@@ -1,14 +1,18 @@
 # builder image
-FROM golang:1.11.5
+FROM golang:1.12.4
+
+ARG HUGO_BASEURL
+ENV HUGO_BASEURL ${HUGO_BASEURL:-https://decred.org}
```

Comment:
This doesn't seem correct. When I run the docker container locally, none of the web resources are loading. See below:

![Screenshot from 2019-05-01 09-43-20](https://user-images.githubusercontent.com/6762864/57010693-f5697700-6bf5-11e9-8e5c-0cbbe4086d36.png)

It works as expected when I remove this ENV var. What was the purpose for adding it?

Reply:
Leftover from testing.  Removed...